Assets/Treasure/TDK/Runtime/Identity/TDK.Identity.cs

```diff
@@ -94,10 +94,12 @@ await TDKServiceLocator.GetService<TDKThirdwebService>().ActiveWallet.CreateSess
 
         private async Task<List<User.Signer>> GetActiveSigners()
         {
-            var activeSigners = await TDKServiceLocator.GetService<TDKThirdwebService>().ActiveWallet.GetAllActiveSigners();
+            var thirdwebService = TDKServiceLocator.GetService<TDKThirdwebService>();
+            var activeSigners = await thirdwebService.ActiveWallet.GetAllActiveSigners();
+            var allAdmins = await thirdwebService.ActiveWallet.GetAllAdmins();
```

Comment:
Can we use `Task.WhenAll` to parallelize these calls?

Reply:
Done! not as pretty as Promise.all but it should work 🙃